cs/form-fields.texy

```diff
@@ -214,6 +214,26 @@ $form->addSubmit('submit', 'Odeslat');
 
 Pokud nechcete validovat formulář při stisknutí tlačítka (například u tlačítek *Zrušit* nebo *Náhled*), můžete ji vypnout pomocí `setValidationScope([])`.
 
+Je v pořádku přidat více než jedno odesílací tlačítko.
```

Comment:
Celé by to mělo přecházet odstavci „Pokud nechcete validovat formulář…“

Reply:
Upravno a opraveno.